app/templates/components/navigation/nav-desktop.html

```diff
@@ -1,4 +1,4 @@
-<ul id="accordion-list" class="h-full p-mobile overflow-scroll max-w-hero">
+<ul id="accordion-list" class="h-full p-mobile overflow-auto max-w-full desktop:max-w-hero">
   {% for section in translations.get("sections", []) %}
     {% with section=section %}
       {% if section.type == "header" %}
```

Comment:
Yeah, I adjusted this structure in #43 so that we can have multiple headers (also in between chart sections) instead of just a single one in the beginning.

Reply:
I agree that it’s not optimal, though! Maybe something like a more nested structure would be more appropriate for it. But it was a small change and unblocked Marco without causing conflicts, so I went with it for now!